parallel_algorithms/P3732/numeric_ranges_algorithms.md

```diff
@@ -56,12 +56,10 @@ R0 is the original draft prepared before the June 2025 Sofia WG21 meeting.  SG1
 
 - Revise non-wording sections
 
-    - Explain `reduce_into` and `transform_reduce_into`
+    - Explain `reduce_into` and `transform_reduce_into`, as well as
+      `sum_into`, `product_into`, and `dot_into` as special cases
 
-    - Show different designs for specifying identity value
-
-- Add `sum_into`, `product_into`, and `dot_into`

```

Comment:
The point of leaving this as a separate bullet point is to indicate that this is a wording change.  It doesn't properly belong to "Revise non-wording sections."  I'd like the change log to highlight changes to the normative design.

Reply:
I see. But then the list should be `reduce_into`, `sum_into`, and `product_into`. There is no proper wording for `dot_into` yet, as far as I can tell.